string robot_name
---
bool success
string robot_info